homeassistant/components/yamaha/media_player.py

```diff
@@ -205,24 +215,38 @@ def __init__(self, name, receiver, source_ignore, source_names, zone_names):
         self._is_playback_supported = False
         self._play_status = None
         self._name = name
-        self._zone = receiver.zone
-        if self.receiver.serial_number is not None:
+        self._zone = zctrl.zone
+        if self.zctrl.serial_number is not None:
             # Since not all receivers will have a serial number and set a unique id
             # the default name of the integration may not be changed
             # to avoid a breaking change.
-            self._attr_unique_id = f"{self.receiver.serial_number}_{self._zone}"
+            # Prefix as MusicCast could have used this
+            self._attr_unique_id = f"{self.zctrl.serial_number}_{self._zone}"
+            _LOGGER.debug(
+                "Receiver zone: %s zone %s uid %s",
+                self._name,
+                self._zone,
+                self._attr_unique_id,
+            )
+        else:
+            _LOGGER.info(
```

Comment:
Idem

Reply:
here I do sort of want to let the user know.

Reply:
where can I use info or should this be warn?

Reply:
Warning or debug